docs/installation.rst

```diff
@@ -5,21 +5,21 @@ Installation
 pip_
 ============
 
-For a traditional Python installation of Planemo, first set up a virtualenv
-for ``planemo`` (this example creates a new one in ``.venv``) and then
+For a traditional Python installation of Planemo, first set up a virtual environment
+for ``planemo`` (this example creates a new one in ``planemo``) and then
 install with ``pip``. Planemo requires pip 7.0 or newer.
 
 ::
 
-    $ virtualenv .venv; . .venv/bin/activate
-    $ pip install "pip>=7" # Upgrade pip if needed.
+    $ python3 -m venv planemo
+    $ source planemo/bin/activate
```

Comment:
`.` is POSIX and more portable than `source` (which is not available in e.g. dash)

```suggestion
    $ . planemo/bin/activate
```